x-pack/plugin/src/test/resources/rest-api-spec/api/async_search.submit.json

```diff
@@ -0,0 +1,226 @@
+{
+  "async_search.submit":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/async-search.html"
+    },
+    "stability":"experimental",
+    "url":{
+      "paths":[
+        {
+          "path":"/_async_search",
+          "methods":[
+            "GET",
+            "POST"
```

Comment:
one more thing I had missed before, here we may want to remove GET? I tend to think that POST is the only method that suits an API that submits something. Was it here only for consistency with search?